### PR TITLE
refactor(log): use LOG_WARNING_F instead of LOG_WARNING (2/3)

### PR DESCRIPTION
**Pull request by WHBANG**
### What problem does this PR solve? 
https://github.com/apache/incubator-pegasus/issues/1305

This patch aim to use LOG_WARNING_F instead of LOG_WARNING. Includes:

Use LOG_WARNING_F instead of LOG_WARNING in meta module.

### CODE REVIEW

File: src/meta/meta_backup_service.cpp [hunk 1]
```diff
@@ -1545,8 +1546,7 @@ void backup_service::modify_backup_policy(configuration_modify_backup_policy_rpc
                 LOG_INFO_F("{}: remove app({}) to policy", cur_policy.policy_name, appid);
                 have_modify_policy = true;
             } else {
-                LOG_WARNING(
-                    "%s: invalid app_id(%d)", cur_policy.policy_name.c_str(), (int32_t)appid);
+                LOG_WARNING_F("{}: invalid app_id({})", cur_policy.policy_name, (int32_t)appid);
```
Comment (empiredan) [inline src/meta/meta_backup_service.cpp:1549]:
```suggestion
                LOG_WARNING_F("{}: invalid app_id({})", cur_policy.policy_name, appid);
```

File: src/meta/meta_backup_service.cpp [hunk 2]
```diff
@@ -1560,9 +1560,9 @@ void backup_service::modify_backup_policy(configuration_modify_backup_policy_rpc
             cur_policy.backup_interval_seconds = request.new_backup_interval_sec;
             have_modify_policy = true;
         } else {
-            LOG_WARNING("%s: invalid backup_interval_sec(%" PRId64 ")",
-                        cur_policy.policy_name.c_str(),
-                        request.new_backup_interval_sec);
+            LOG_WARNING_F("{}: invalid backup_interval_sec({})",
+                          cur_policy.policy_name.c_str(),
```
Comment (empiredan) [inline src/meta/meta_backup_service.cpp:1564]:
```suggestion
                          cur_policy.policy_name,
```

File: src/meta/server_state.cpp [hunk 3]
```diff
@@ -1796,15 +1792,14 @@ void server_state::downgrade_primary_to_inactive(std::shared_ptr<app_state> &app
...
             return;
         } else {
-            LOG_WARNING("gpid(%d.%d) is syncing another request with remote, cancel it due to the "
-                        "primary(%s) is down",
-                        pc.pid.get_app_id(),
-                        pc.pid.get_partition_index(),
-                        pc.primary.to_string());
+            LOG_WARNING_F("gpid({}) is syncing another request with remote, cancel it due to the "
+                          "primary({}) is down",
+                          pc.pid.get_app_id(),
```
Comment (empiredan) [inline src/meta/server_state.cpp:1801]:
```suggestion
                          pc.pid,
```

**Review by empiredan — APPROVED** [review]

**Comment by WHBANG** [comment]
@neverchanje hi, can you check this pr again, thank you very much.

**Review by acelyc111 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**